### PR TITLE
Update C920s README

### PR DESCRIPTION
**Pull request by awickham10**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
No additions - just updating the README for Iakabos' Logitech C920 mount mod to include a BOM and assembly instructions.

#### How was it tested? 
N/A

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
N/A

#### Further notes
N/A

### CODE REVIEW

File: printer_mods/Iakabos/C920_mount/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
+
+Qty | Item
+:-- | :--
+1 | Logitech C920/C920s Webcam
+2 | M3 t-nuts
+2 | M3x8 SHCS
+
+## Instructions
+1. Disassemble webcam using instructions [seen here](https://www.ifixit.com/Guide/Logitech+C920+Webcam+Disassembly/115077) stopping after step 5
+2. Remove the metal arms can be removed from the factory supplied mount
```
Comment (KiloQubit) [inline printer_mods/Iakabos/C920_mount/README.md:14]: This sentence doesn't make sense.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your update!

Did you check with the mod author (@Iakabos) about making updates to this mod?  I'm going to need approval from the original mod author to merge this.

File: printer_mods/Iakabos/C920_mount/README.md (hunk 1 above)
Comment (Iakabos) [reply printer_mods/Iakabos/C920_mount/README.md:14]: Agreed. Would suggest changing it to say "Remove the metal arms from the factory supplied mount"

**Comment by Iakabos** [comment]
Other than the suggested wording change for step 2, these changes look good to me

**Comment by awickham10** [comment]
I submitted the change - some sort of brain fart typo on my part.

Didn't mean any disrespect @Iakabos just was going through the mod for my own printer and thought I'd help out where I struggled.

**Review by KiloQubit — APPROVED** [review]
Thanks for the update!

**Comment by Iakabos** [comment]
No problem @awickham10  I appreciate the improvements

**Pull request merged**